resources/asciidoctor/spec/spec_helper.rb

```diff
@@ -65,12 +65,17 @@ def internal_convert(input, convert_logger, extra_attributes)
 #   logs             - lines logged
 RSpec.shared_context 'convert with logs' do
   let(:convert_logger) { Asciidoctor::MemoryLogger.new }
-  let!(:converted) do
-    # We use let! here to force the conversion because it populates the logger
+  let(:converted) do
+    # Using let! here would stop us from having to explicitly evaluate
+    # `converted` in the let for `logs` but it'd cause `converted` to be
+    # evaluated before `before(:example)` blocks
```

Comment:
This is not quite related but I just looked up what the rspec ordering was exactly for `before`, `let!`, etc and I found this https://relishapp.com/rspec/rspec-core/v/3-8/docs/configuration/overriding-global-ordering
I've never heard about that before, or used it, but I thought it was interesting and would possibly to useful to you at some point.

Reply:
I'll take a look at it!